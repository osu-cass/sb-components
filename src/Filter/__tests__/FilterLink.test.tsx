import * as React from "react";
import { mount } from "enzyme";
import "jsdom-global/register";
import { FilterLink } from "@src/index";

const wrapper = mount(<FilterLink filterId="sb-filter-id" />);

it("filter link test", () => {
  wrapper.simulate("click");
  expect(wrapper).toMatchSnapshot();
});
