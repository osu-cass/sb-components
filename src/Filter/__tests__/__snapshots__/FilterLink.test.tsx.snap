// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`filter link test 1`] = `
<Component
  filterId="sb-filter-id"
>
  <a
    className=" filter-jump-link btn btn-gray"
    onClick={[Function]}
    role="button"
  >
    <span
      className="fa fa-filter"
    />
    Jump to Filter
  </a>
</Component>
`;
